doc/get_started_on_minikube.md

```diff
@@ -0,0 +1,56 @@
+# Get started on minikube
+
+This document describes how to deploy Chaos Mesh in Kubernetes on your laptop (Linux or macOS) using minikube.
+
+## Prerequisites
+
+Before deployment, make sure [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) 
+is installed on your local machine.
+
+## Step 1: Setup the Kubernetes environment
+
+Take the following steps to set up the local Kubernetes environment:
+
+1. Start a `minikube` kubernetes cluster. Make sure you have installed [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+
+   ```bash
+   minikube start --kubernetes-version v1.15.0 --cpus 4 --memory "8192mb" # we recommend that you allocate enough RAM (more than 8192 MiB) to the VM
+   ```
+
+2. Install Helm
+
+   ```bash
+   curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | bash
+   helm init
```

Comment:
It seems that `helm init` is not available for 3.x .

Reply:
Thanks for your reviewing, I will fix it

Reply:
This scripts will install helm v2, so `helm init` is ok

Reply:
Yes. I miss that. Maybe we should change the title to `Intall Helm2` ?